src/hooks/useInput/index.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+function useInput(initialValue) {
+  const [value, setValue] = useState(initialValue);
+
+  return {
+    value,
+    setValue,
+    reset: () => setValue(""),
+    bind: {
+      value,
+      onChange: (event) => {
+        setValue(event.target.value);
+      },
+    },
+  };
+}
+
+export default useInput;
```

Comment:
What could be the new name? I think I can use setValue("") instead reset, where I use this hook. So I can remove the reset function to avoid confusion.

Reply:
You could either export `setValue` directly to have access to the setter of the custom hook, or if you'd like to keep the reset function, perhaps changing it to `resetValue` (or something like that)

Reply:
Thanks, I have renamed the function for resetValue.